course/auth.py

```diff
@@ -911,7 +907,9 @@ def __init__(self, *args, **kwargs):
                     "<b>Once %(submitted_or_verified)s, it cannot be "
                     "changed</b>.")
                 % {"submitted_or_verified":
-                    EDITABLE_INST_ID_BEFORE_VERIFICATION
+                    getattr(settings,
+                            "RELATE_EDITABLE_INST_ID_BEFORE_VERIFICATION",
+                            True)
```

Comment:
Could you make a function to get that setting? Otherwise the default value is repeated, which could lead to inconsistencies.

Reply:
That's reasonable. The realization before is using a constant, that brought trouble for tests (because overiding settings won't work for testing different configurations), but a function will be ok. Will update.

Reply:
Updated.